pgjdbc/src/main/java/org/postgresql/core/Parser.java

```diff
@@ -1446,4 +1443,43 @@ private int getMatchedPosition(char[] sql, int pos) {
       return 0;
     }
   }
+
+  static final class IntCollection {
```

Comment:
I guess it should be `IntList` to enforce it keeps the order of elements, and it could probably move to a top-level.
`IntList` would be helpful for https://github.com/pgjdbc/pgjdbc/pull/1946